share/smack/top.py

```diff
@@ -420,13 +420,21 @@ def verify_bpl(args):
     command += ["/maxStaticLoopBound:%d" % args.loop_limit]
     command += ["/recursionBound:%d" % args.unroll]
 
+  elif args.verifier == 'symbooglix':
+    command = ['symbooglix']
+    command += [args.bpl_file]
+    command += ["--file-logging=0"]
+    command += ["--entry-points=%s" % ",".join(args.entry_points)]
+    command += ["--timeout=%d" % args.time_limit]
+    command += ["--max-depth=%d" % args.unroll]
```

Comment:
Got it. Yeah, most other Boogie checkers (if not all) define loop/recursion bound as the parameter for bounding executions. Ultimately, we would like to run Symbooglix on all SVCOMP benchmarks and do a comparison with e.g. Corral. Currently, it would be hard/unfair to compare them given that we would have to unsure that we are bounding the exploration in the same way (or at least very similar).
Now, once we start running Sumbooglix on SVCOMP, is there a chance you would be interested in implementing support for specifying loop bounds?